pkg/flags/flags.go

```diff
@@ -38,6 +38,7 @@ const (
 	flagDryRunName              = "dry-run"
 	flagFailFast                = "fail-fast"
 	flagDisableGracefulTeardown = "disable-graceful-teardown"
+	flagKubeContext             = "kube-context"
```

Comment:
I think it would be nice to have a document added in the examples indicating how to use this flag.